### ISSUE TITLE
2D Views: Label Capping Bottoms

### ISSUE THREAD

**Issue by TedCC1209** [CCDevNelson/Snap-Updates#262]
Need to add a label for capping bottoms that also contains its depth. The hash mark would come from the bottom center of the capping bottom, point down and left to the annotation that says "Capping Bottom [depth in inches]".

**Comment by AntonioGloria** [comment]
Hi @TedCC1209,
The labels for the capping bottoms are done.
![Capping_Bottom_Label](https://user-images.githubusercontent.com/74732175/105426162-4f06a600-5c10-11eb-90b5-62c2063a1900.png)
Here's the link for this feature: https://github.com/nailsonlandim/SNaP-Dev/tree/issue_%23262-label-capping-bottoms

**Comment by TedCC1209** [comment]
Your example looks good Antonio, thank you. @kategodfrey @sarahhightower @JBTattersall can you guys test this and give us your feedback please? Thanks, Ted

**Comment by sarahhightower** [comment]
Looks great. Thanks! 

![Screenshot 2021-01-22 090051](https://user-images.githubusercontent.com/72047269/105514144-65544680-5c90-11eb-88dc-3ee14b2a279b.png)

**Comment by kategodfrey** [comment]
I tested this twice and it doesn't show up... I will try again later this evening. I think what could have happened was that when I opened Nailson's Snap to work in, the pop-up to update to version 1.2.2. came up and I pressed it without thinking... But like I said, I will try again later this evening when I get back to my computer.

**Comment by TedCC1209** [comment]
@AntonioGloria @nailsonlandim  Can we have your repository updated to have all the latest merges to the SNaP-Dev master branch? Even the master branch of Nailson's repository asks to update to v1.2.2 when I open it. Then we can test again after that. Thanks, Ted

**Comment by nailsonlandim** [comment]
Ok, I will fix that.

**Comment by nailsonlandim** [comment]
Fixed, 

You can look at it again

let me know if the problem persists.

Thanks,
Nailson

**Comment by kategodfrey** [comment]
I was able to get it to appear. The first time I did it, it did not appear... I think it is because I updated to 1.2.3? This last time I did not update and was able to get it to appear 👍

![image](https://user-images.githubusercontent.com/63115502/105879357-4d95fe80-5fbf-11eb-99a8-787f06dcfa45.png)

**Comment by TedCC1209** [comment]
Yes, @ClassyTeddy @nailsonlandim @JJanneh @AntonioGloria all of the outstanding branches you have been working on with changes that have not yet been merged with the Master look like they need to be updated with v1.2.3 and the most recent changes to the master branch.

**Comment by TedCC1209** [comment]
I think this is good to merge, again though I am worried about the branch as it relates to the latest fixes and updates. @ClassyTeddy if @AntonioGloria needs to update this branch before merging please advise. Thanks, Ted

**Comment by ClassyTeddy** [comment]
Updating the branch through SNaP will remove all of the changes made. Merging now should be fine.
